todo/__init__.py

```diff
@@ -1,10 +0,0 @@
-"""
```

Comment:
Oops - you've accidentally wiped out the __init__.py - I can no longer install the app. Please restore this file.